docker/dev/ubuntu/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Develop Orion Context Broker Ubuntu-Latest
+FROM ubuntu:latest
```

Comment:
yes, we are going to need the CentOS one as well. Maybe you can contribute it, taking as input the work done at https://registry.hub.docker.com/u/bitergia/fiware-orion/dockerfile/